dialog/lib/dialog.ts

```diff
@@ -349,7 +349,11 @@ export class Dialog extends LitElement {
     }
     // Avoids dispatching initial state.
     const shouldDispatchAction = changed.get('open') !== undefined;
-    this.performTransition(shouldDispatchAction);
+
+    // Avoids performing unnecessary transition
+    if (changed.get('open') !== undefined && changed.get('open') !== this.open) {
```

Comment:
`shouldDispatchAction` is the same value as the first condition in this if block. Is that intentional? Would be the same as

```ts
if (changed.get('open') !== undefined && changed.get('open') !== this.open) {
  this.performTransition(true);
}
```

Are there other places that `performTransition` is called with a false value?